content/docs/studio/user-guide/model-registry/register-version.md

```diff
@@ -0,0 +1,49 @@
+# Version models
+
+You can register new versions of registered models by specifying the Git commit
+which corresponds to the new version.
+
+To register a new version of a model, Iterative Studio uses [GTO] to create an
+[annotated Git tag][git tag] with the specified version number. Refer the [GTO
+docs][gto] to see the exact format of the Git tag.
+
+You can register versions using the [GTO] command line interface (CLI). To
+register versions using Iterative Studio, watch this tutorial video or read on
+below:
+
+https://youtu.be/eA70puzOp1o
```

Comment:
- [ ] Make video public (and change link to https://www.youtube.com/watch?v=eA70puzOp1o ?)